src/IssueViz/Helpers/IAnalysisSeverityToVsSeverityConverter.cs

```diff
@@ -66,4 +86,14 @@ public __VSERRORCATEGORY Convert(AnalysisIssueSeverity severity)
             }
         }
     }
+
+    public static class AnalysisSeverityToVsSeverityConverterExtensions
```

Comment:
Instead of adding this as a extension method wouldn't it make more sense to add it as public method to the IAnalysisSeverityToVsSeverityConverter and make the other two private?

Reply:
I can see your point, but I would need to change all of the tests. If you think it's worth it I'll do it

Reply:
let's keep it for now but create a hardening ticket for the future.